.travis.yml

```diff
@@ -2,8 +2,11 @@
 
 language: objective-c
 os: osx
+# Add a TEST_DIR=[directory] row for each library (or group of libraries) to test
+# Each will be run as an independent, hermetic test (in parallel when resources permit)
+# on Traivs.
 env:
-  - USING_OS=osx USING_XCODE=6.1
+  - TEST_DIR=gson
```

Comment:
I think the directory names should follow the package name convention, replacing `:` with `-`. In this case:

com.google.code.gson-gson